modules/effects/arrow-effects-rx2/src/main/kotlin/arrow/effects/SingleK.kt

```diff
@@ -58,7 +68,7 @@ data class SingleK<A>(val single: Single<A>) : SingleKOf<A>, SingleKKindedJ<A> {
 
     fun <A> async(fa: Proc<A>): SingleK<A> =
       Single.create { emitter: SingleEmitter<A> ->
-        fa { either: Either<Throwable, A> ->
+        fa(IOConnection()) { either: Either<Throwable, A> ->
```

Comment:
Same about emitter

Reply:
`IOConnection` is not exposed through Async anymore, mimicking cats. Further explanations in a comment in the end.